src/kixi/datastore/web_server.clj

```diff
@@ -290,32 +299,54 @@
    metrics
    {:id :schema-id-entry
     :methods
-    {:get {:produces "application/transit+json"
+    {:get {:produces "application/json"
            :response
            (fn [ctx]
              (let [id (get-in ctx [:parameters :path :id])]
                (ss/fetch-with schemastore {::ss/id id})))}}}))
 
-(defn add-ns-to-keywords
+(defn add-ns-to-keys
   ([ns m]
    (letfn [(process [n]
              (if (= (type n) clojure.lang.MapEntry)
                (clojure.lang.MapEntry. (keyword (namespace ns) (name (first n))) (second n))
                n))]
      (walk/prewalk process m))))
 
+(defn map-every-nth [f coll n]
+  (map-indexed #(if (zero? (mod %1 n)) (f %2) %2) coll))
+
+(defn keywordize-values
+  [m]
+  (let [keys-values-to-keywordize [::ss/name]
+        m' (reduce
+            (fn [a k]
```

Comment:
Hmmm, i thought i didn't need that bit any more, changed stuff so i could just use conform. Single char params are a good topic for chat. I find myself using them more, especially in cases like this, in a reducer the first param is always the accumulator, which can just be 'a' to me now, after going through an 'acc' stage. 'k' is always 'key' but key is a keyword, so just have to shorten it!